src/cloud/internal/cloudservice.cpp

```diff
@@ -165,7 +169,21 @@ void CloudService::onUserAuthorized()
     m_refreshToken = m_oauth2->refreshToken();
 
     saveTokens();
-    downloadUserInfo();
+
+    if (downloadUserInfo() == RequestStatus::Ok) {
+        m_onUserAuthorizedCallback();
```

Comment:
I believe need to add `IF_ASSERT_FAILED(m_onUserAuthorizedCallback)`

Reply:
m_onUserAuthorizedCallback can be invalid (see CloudService::signIn()). This callback is needed when you need something to do right after authorization (for example, uploading of the score will continue if a user wasn't authorized when they pressed on "Upload to MuseScore.com")

Reply:
Then you just need to check `if (m_onUserAuthorizedCallback)`, otherwise it will crash :)

Reply:
fixed